app/assets/stylesheets/ama_layout/layout/helper-classes.scss

```diff
@@ -218,6 +218,23 @@
   background-color: rgba(255,255,255,0.6);
 }
 
+.vr{
+  &--desktop{
```

Comment:
so I think this desktop modifier isn't really necessary because the vertical rule is for desktop. So we don't need a modifier to say that. We could just have the styles that are currently under the modifier apply to .vr